compiler/rustc_lint_defs/src/lib.rs

```diff
@@ -294,6 +286,10 @@ impl Level {
             _ => None,
         }
     }
+
+    pub fn min(self, other: Self) -> Self {
```

Comment:
hm, this seems not much more useful than just using `cmp::min` 🤔

Reply:
`cmp::min` needs `Ord`, too